lib/aws/amplify/signoutButton/SignoutButton.tsx

```diff
@@ -0,0 +1,14 @@
+import {useAuthenticator} from "@aws-amplify/ui-react-native";
```

Comment:
button should be a generic component it shouldn't be related to aws and you should pass onPress as a prop